src/TimeSeries/widget/components/__tests__/NVD3LineChart.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { ReactWrapper, mount, shallow } from "enzyme";
+import { DOM, HTMLAttributes, createElement } from "react";
+
+import { NVD3LineChart, Nvd3LineChartProps, configureComponents, isPlainObject } from "../NVD3LineChart";
+
+describe("NVD3LineChart", () => {
+
+    const renderChart = (props?: Nvd3LineChartProps) => shallow(createElement(NVD3LineChart, props));
+    const renderFull = (props?: Nvd3LineChartProps) => mount(createElement(NVD3LineChart, props));
```

Comment:
Alright, we will only use the mount when testing the lifecycle methods componentDidMount and componentWillUnmount